### PR TITLE
Add text classification with Fluid.

### LINKED ISSUE #605: Add text classification with Fluid.
num_pass: 30
batch_size: 4
use_gpu: False

Time cost:

CPU: Intel(R) Xeon(R) CPU E5-2620 v2 @ 2.10GHz
Train time: 7252.858182s
commit id:  fb1a0dfb157032f3eb7cdabcb445a77656ef2498

CPU: Intel(R) Core(TM) i5-5257U CPU @ 2.70GHz
Train time: 5022.535137s
commit id: 4fb3c676a8f229684f0369d5e2e0d549c97565a8

### PR DESCRIPTION
**Pull request by peterzhang2029**
resolve https://github.com/PaddlePaddle/models/issues/605

### CODE REVIEW

File: fluid/text_classification/train.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    res.set_lod([lod])
+    return res
+
+
+# Load the dictionary.
+def load_vocab(filename):
+    vocab = {}
+    with open(filename) as f:
+        wid = 0
+        for line in f:
```
Comment (lcy-seso) [inline fluid/text_classification/train.py]: for idx, line in enumerate(f)

File: fluid/text_classification/train.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+    inference_program = fluid.default_main_program().clone()
+    with fluid.program_guard(inference_program):
+        test_target = accuracy.metrics + accuracy.states
+        inference_program = fluid.io.get_inference_program(test_target)
+
+    # The training data set.
+    train_reader = paddle.batch(
+        paddle.reader.shuffle(
+            paddle.dataset.imdb.train(word_dict), buf_size=5000),
```
Comment (lcy-seso) [inline fluid/text_classification/train.py]: 把这个buf_size改到5w+，把数据一次性都读进来再测试一个pass的时间。

File: fluid/text_classification/train.py (hunk 1 above)
Comment (peterzhang2029) [reply fluid/text_classification/train.py]: Done

File: fluid/text_classification/train.py (hunk 2 above)
Comment (peterzhang2029) [reply fluid/text_classification/train.py]: Done, the result is in the bottom of this issue  https://github.com/PaddlePaddle/models/issues/607

**Review by lcy-seso — APPROVED** [review]

**Pull request merged**